service/src/main/kotlin/fi/espoo/evaka/emailclient/EvakaEmailMessageProvider.kt

```diff
@@ -319,7 +323,9 @@ $unsubscribeEn
                     else Triple("tiedote", "allmänt meddelande", "bulletin")
             }
         return EmailContent.fromHtml(
-            subject = "Uusi $typeFi eVakassa / Nytt $typeSv i eVaka / New $typeEn in eVaka",
+            subject =
+                if (showMessageSubjectInEmailSubject) thread.title
```

Comment:
This needs HTML escaping (with `HtmlEscape.escapeHtml5`), because it's directly embedded as raw HTML and the data comes from a user and is considered untrusted